### ISSUE TITLE
Fine tune Kinetics on HMDB51 split2 - Can't achieve same results

### ISSUE THREAD

**Issue by Esaada** [cypw/PyTorch-MFNet#30]
Hi, as mentioned in the paper, and in most of the papers in the action recognition field, the result is averaged on the 3 splits. because only split1 finetuning shared, I'm trying to fine tune my self, but I can't get the same results, the training is very "noisy", I'm using the pretrained model on kinetics that was uploaded, is there anything else I need to keep my attention on?
I'm using your environment of course. (btw in the annotation files, others, I believe means validation set?)
Thanks!
